presentation/src/main/res/layout/fragment_forgot_password.xml

```diff
@@ -5,127 +5,132 @@
         android:id="@+id/button_back"
-        style="@style/Back_arrow"
-        android:layout_width="24dp"
-        android:layout_height="24dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <ImageView
-        android:id="@+id/logo_safetyheads"
-        style="@style/Logo_safetyheads"
-        android:layout_width="128dp"
-        android:layout_height="96dp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/button_back"
-        android:layout_marginTop="@dimen/size" />
-
-    <ImageView
         android:layout_width="wrap_content"
-        android:layout_height="0dp"
-        android:layout_marginEnd="@dimen/size"
-        android:src="@drawable/h2_view_line_63dp"
-        app:layout_constraintBottom_toBottomOf="@+id/tv_forgot_pass_text"
-        app:layout_constraintEnd_toStartOf="@+id/tv_forgot_pass_text"
-        app:layout_constraintTop_toTopOf="@+id/tv_forgot_pass" />
-
-    <TextView
-        android:id="@+id/tv_forgot_pass"
-        style="@style/Text_comforta_bold_18sp"
-        android:layout_marginStart="10dp"
-        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/size_4x"
-        android:textColor="@color/s_60"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/logo_safetyheads"
-        android:text="@string/forgot_password_without_underline" />
+        app:layout_constraintTop_toTopOf="parent"/>
 
-    <TextView
-        android:id="@+id/tv_forgot_pass_text"
-        style="@style/Poppins_regular_green_10sp"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/size_0.25x"
-        android:layout_marginStart="10dp"
-        android:text="@string/forgot_password_text"
-        android:textColor="@color/n_120"
-        android:textSize="14sp"
+        android:layout_margin="@dimen/size_3x"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tv_forgot_pass" />
+        app:layout_constraintTop_toBottomOf="@id/button_back"
+        app:layout_constraintEnd_toEndOf="parent">
 
-    <TextView
-        android:id="@+id/email_address"
-        style="@style/Text2"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/size_3x"
-        android:text="@string/e_mail"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tv_forgot_pass_text" />
+        <ImageView
+            android:id="@+id/logo_safetyheads"
+            style="@style/Logo_safetyheads"
+            android:layout_width="128dp"
```

Comment:
can't we make those 0dp and somehow align it in the layout so it looks good and it is not hardcoded?